main.go

```diff
@@ -63,7 +63,7 @@ func main() {
 	args := flag.Args()
 	py.MustGetModule("sys").Globals["argv"] = pysys.MakeArgv(args)
 	if len(args) == 0 {
-		cli.RunREPL()
+		cli.RunREPL(version, commit, date)
```

Comment:
@ncw 
Is there any reason not to move
```go
fmt.Printf("Python 3.4.0 (%s, %s)\n", commit, date)
fmt.Printf("[Gpython %s]\n", version)
fmt.Printf("- os/arch: %s/%s\n", runtime.GOOS, runtime.GOARCH)
fmt.Printf("- go version: %s\n", runtime.Version())
```
By this way, we can avoid to pass `version, commit, date` to exectute `RunREPL`

Reply:
I guess that would probably be OK!

Reply:
@kislenko-artem Please follow the comment I left! Thanks!

Reply:
@corona10 sorry I can't figure out what I have to do. Output of version of Python was inside Repl function. It was just - fmt.Printf("Gpython 3.4.0\n") I added some attributes. Where output of version should be? before cli.RunREPL and inside block if len(args) == 0 ? Is it OK?

Reply:
@kislenko-artem I mean move out the fmt.Printf from RunREPL,
and do it on main.go :)